device/device.go

```diff
@@ -252,3 +277,12 @@ func (d *Device) AddInterface(astarteInterface interfaces.AstarteInterface) erro
 func (d *Device) RemoveInterface(astarteInterface interfaces.AstarteInterface) {
 	delete(d.interfaces, astarteInterface.Name)
 }
+
+func getMaxInflightMessages(defaultValue int) int {
+	if valueStr := os.Getenv("MAX_INFLIGHT_MESSAGES"); valueStr == "" {
```

Comment:
I would expect this to be something you set on the device itself (like `IgnoreSSLErrors`) instead of an environment variable